src/App.js

```diff
@@ -1,27 +1,11 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
```

Comment:
- [ ] Your `App.js` component should be class-based, Please update that 

![image](https://user-images.githubusercontent.com/75126481/153275258-2ae2676d-4b62-4a86-94b1-107aa895c026.png)